packages/bot-strategy-runner/test/ConfigBuilder.ts

```diff
@@ -35,7 +35,7 @@ const minimumRequiredConfigKeys = [
   "pollingDelay",
 ];
 describe("buildGlobalWhitelist", async function () {
-  it("Correctly fetches global whitelist", async function () {
+  it("Correctly fetches global whitelist", async function (this: any) {
```

Comment:
@chrismaree not sure why this tsc error wasn't caught before but i needed to fix this

Reply:
that's strange...not sure why. it must be related to your package change?